tests/csm/rest/test_iam_users.py

```diff
@@ -1932,3 +1932,185 @@ def test_38094(self):
                                      self.rest_resp_conf[4099]['empty key'][1]
                                      , "Response message check failed")
         self.log.info("##### Test completed -  %s #####", test_case_name)
+
+    @pytest.mark.csmrest
+    @pytest.mark.lc
+    @pytest.mark.cluster_user_ops
+    @pytest.mark.parallel
+    @pytest.mark.tags('TEST-39029')
+    def test_39029(self):
+        """
+        Add-remove random capabilities
+        """
+        test_case_name = cortxlogging.get_frame()
+        self.log.info("##### Test started -  %s #####", test_case_name)
+        self.log.info("[START] Creating IAM user")
```

Comment:
Can you just add log messages of steps present in Test Design for better understanding of test functionality?

Reply:
Added